### ISSUE TITLE
contrib-type is not being defined after saving

### ISSUE THREAD

**Issue by fabiobatalha** [substance/texture#402]
Not sure if it is a bug or a expected result at this moment, but the last author was included in the document and after saving it, the contrib-group was not defined for the new author.

``` xml
      <contrib-group id="contrib-group-2" content-type="author">
        <contrib id="contrib-1" contrib-type="person">
          <name>
            <surname>Lage</surname>
            <given-names>João Gabriel Batista</given-names>
          </name>
          <xref ref-type="aff" rid="aff1"/>
        </contrib>
        <contrib id="contrib-2" contrib-type="person">
          <name>
            <surname>Gutierrez</surname>
            <given-names>Paulo Sampaio</given-names>
          </name>
          <xref ref-type="aff" rid="aff1"/>
        </contrib>
        <contrib id="contrib-f11ca13b4d023df633da285a76aca125">
          <name>
            <surname>Batalha</surname>
            <given-names>Fabio</given-names>
          </name>
        </contrib>
      </contrib-group>
```

**Comment by michael** [comment]
Thank you for reporting. We'll work on a fix.

**Comment by michael** [comment]
Fixed via #407.
